models/api.models.js

```diff
@@ -16,9 +17,29 @@ const fetchAllTopics = () => {
 const fetchAllEndpoints = () => {
   return fs.readFile("./endpoints.json").then((results) => {
     const parsedResults = JSON.parse(results);
-    // console.log(parsedResults, "results here");
     return parsedResults;
   });
 };
 
-module.exports = { fetchAllTopics, fetchAllEndpoints };
+const fetchArticleById = (id) => {
+  if (id === undefined || !typeof id === "number") {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
+  }
```

Comment:
There is no need to do this if statement. If the id is not a number, PSQL will throw an error and you can handle it that way. The id would never undefined as that would be a different endpoint altogether